<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingList</class>
 <widget class="QFrame" name="BookingList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>435</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="BookingPlanning" name="planning">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNewBooking">
         <property name="text">
          <string>Ajouter une r√©servation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEditSelectedBooking">
         <property name="text">
          <string>Modifier...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="BookingForm" name="bookingForm">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BookingTableView" name="bookingsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookingTableView</class>
   <extends>QFrame</extends>
   <header>Booking.BookingTableView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookingPlanning</class>
   <extends>QFrame</extends>
   <header>BookingPlanning</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookingForm</class>
   <extends>QFrame</extends>
   <header>BookingForm</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../loteliQt.qrc"/>
 </resources>
 <connections/>
</ui>
